preposterous-kumquat/preposterous-kumquat
# Connect front and Back ends

#### What's this PR do?

Debug backend with client side login/signup/sessions

Remove bundle.js from git tracking
#### Where should the reviewer start?
#### How should this be manually tested?

first of all, get your backend running

run webpack then point the browser to localhost:3000

signup/login

notice changes to db and server logging
#### Any background context you want to provide?

I made the dependency script for jquery into a cdn... that might be an issue on my particular machine... gonna debug that this weekend
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Questions:
